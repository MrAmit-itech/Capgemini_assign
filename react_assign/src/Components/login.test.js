import Login from "./login"

import {screen,render,fireEvent} from "@testing-library/react"

test('testing the click function',()=>{
    const Handleclick = jest.fn()
    
    const {container} = render(<Login onClick={Handleclick} >{"my_button"}</Login>)
    expect(container).toMatchSnapshot()

    fireEvent.click(screen.getByText('my_button'))

    expect(Handleclick).toHaveBeenCalledTimes(1)
})