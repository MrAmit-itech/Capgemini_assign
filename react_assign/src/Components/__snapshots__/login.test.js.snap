// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the click function 1`] = `
<div>
  this is the login page
  <button>
    my_button
  </button>
</div>
`;
